pkg/config/samples/default.dusk.toml

```diff
@@ -153,6 +153,8 @@ accumulatorWorkers = 4
 defaultlocktime = 250000
 # default amount, in whole units of DUSK, to send for consensus transactions.
 defaultamount = 5
+# the timeout for consensus step timers
+consensustimeout = 1
```

Comment:
Default `ConsensusTimeout` used to be 5 sec (as per `config.ConsensusTimeOut`).  Why do you change it to 1 sec?

Reply:
Good catch, will fix it in #579